pkg/template/license_context.go

```diff
@@ -81,12 +84,23 @@ func (ctx licenseCtx) licenseDockercfg() string {
 	auth := fmt.Sprintf("%s:%s", ctx.License.Spec.LicenseID, ctx.License.Spec.LicenseID)
 	encodedAuth := base64.StdEncoding.EncodeToString([]byte(auth))
 
+	proxyRegistryDomain := "proxy.replicated.com"
```

Comment:
Check out `ProxyEndpointFromLicense`.  It works with staging and local env too

Reply:
Will hold on this and utilize the new func added here: https://github.com/replicatedhq/kots/pull/3447/files#diff-584116004557104b2833db1d8c0cd98df3410a21ee778aba079974cd92a1ef94R56